solutions/devsprint-denis-vieira-1/app/src/main/java/com/devpass/githubapp/presentation/RepositoryListActivity.kt

```diff
@@ -31,14 +34,21 @@ class RepositoryListActivity : AppCompatActivity() {
 
         callback.enqueue(object : Callback<List<Repository>> {
             override fun onFailure(call: Call<List<Repository>>, t: Throwable) {
-                Toast.makeText(baseContext, t.message, Toast.LENGTH_SHORT).show()
+                showToast(t.message.toString())
```

Comment:
suggestion: o que acha de mostrar uma mensagem amigável para o usuário e permitir que ele refaça o request invés de apenas mostrar o erro no Toast ?

Você pode utilizar da Snackbar também para esses casos, ela permite que vc adc uma action, segue link: https://developer.android.com/training/snackbar/action#:~:text=You%20can%20add%20an%20action,action%20by%20pressing%20the%20button.